### ISSUE TITLE
Panel: flaky e2e test matchImageSnapshot

### ISSUE THREAD

**Issue by iterox** [dso-toolkit/dso-toolkit#2748]
### Aanleiding

Met enige regelmaat faalt 

```
{
  test: 'Panel - should matchImageSnapshot (cypress/e2e/panel.cy.ts)',
  error: 'Error: Image size (1000x324) different than saved snapshot size (1000x323).\n' +
    'See diff for details: /usr/src/app/storybook/cypress/snapshot-diff/cypress/e2e/panel.cy.ts/Panel -- should matchImageSnapshot.diff.png\n' +
    '    at Context.eval (webpack://dso-storybook/../node_modules/@simonsmith/cypress-image-snapshot/command.js:68:0)'
}
```

In dit issue gaan we dat afwijkende hoogte adresseren en oplossen.

**Comment by tfrijsewijk** [comment]
```js
{
  test: 'Panel - should matchImageSnapshot (cypress/e2e/panel.cy.ts)',
  error: 'Error: Image size (1000x324) different than saved snapshot size (1000x323).\n' +
    'See diff for details: /usr/src/app/storybook/cypress/snapshot-diff/cypress/e2e/panel.cy.ts/Panel -- should matchImageSnapshot.diff.png\n' +
    '    at Context.eval (webpack://dso-storybook/../node_modules/@simonsmith/cypress-image-snapshot/command.js:68:0)'
}
```

The error is telling us that the baseline is 1px shorter than the snapshot. And if you look closely, you can see that the baseline is indeed 1px shorter:

![Panel -- should matchImageSnapshot diff](https://github.com/user-attachments/assets/a0296d29-17b1-4464-a1d9-1989829c77ff)

It didn't happen all the time. The problem appeared to be async related.

## TLDR
In our test markup we use `<i>` to render some cursive text. Removing that element prevents the jump.

## The tour

The first thing I noticed loading a variable font for italic was the last request in my Network tab:

![image](https://github.com/user-attachments/assets/2653ca38-caf6-4a5c-bac4-eda11df9e49e)

And indeed, I was rendering some cursive text:

![image](https://github.com/user-attachments/assets/43615658-7ae8-418c-ba8d-2652ace2b51e)

### Test setup

Testing this was horrible. The flaky test only occured in CI, which ran headless. There is no browser to debug in and logs (either `console.log()` or `cy.log()`) aren't printed to the terminal in CI.

So I created the following debug setup:

```ts
describe("Panel", () => {
  const heights = [];

  beforeEach(() => {
    cy.visit("http://localhost:45000/iframe.html?id=core-panel--default");

    cy.window().then((win) => {
      (win as any).heightsTimer = setInterval(() => {
        const clientHeight = win.document.querySelector("dso-panel")?.clientHeight;

        if (!heights.includes(clientHeight)) {
          heights.push(clientHeight);
        }
      }, 5);
    });
  });

  it("should matchImageSnapshot", () => {
    cy.get("dso-panel.hydrated").matchImageSnapshot();
  });

  afterEach(() => {
    cy.window().then((win) => {
      const { heightsTimer } = win as unknown as { heightsTimer: number };

      win.clearInterval(heightsTimer);

      cy.log("Timer stopped");
    });

    cy.task("log", `Heights: (${heights.length}) ${JSON.stringify(heights)}`);
  });
});
```

Running this several times gave me the following results:

```
Heights: (3) [0,290,289]
Heights: (3) [0,290,289]
Heights: (3) [0,290,289]
Heights: (3) [0,290]
Heights: (3) [0,290,289]
```

Only during the fourth build the snapshot test was passing.

When I removed the `<i>` elements and running it again:

```
Heights: (3) [0,290]
Heights: (3) [0,290]
Heights: (3) [0,290]
Heights: (3) [0,290]
Heights: (3) [0,290]
```

Bingo. Some further tests:

* Setting `font-style: italic` on the `<li>` elements only gives two heights, build always passes ✅
* Replacing `<i>` with `<span style="font-style: italic">` gives the same flaky results ❌
